bluepysnap/network.py

```diff
@@ -38,10 +38,6 @@ def _get_populations(self, cls, config):
         for file_config in config:
             storage = cls(file_config, self._circuit)
             for population in storage.population_names:  # pylint: disable=not-an-iterable
-                if population in res:
-                    raise BluepySnapError(
-                        f"Duplicated {self.__class__.__name__} population: '{population}'"
-                    )
```

Comment:
Should this be allowed?

Reply:
No, not at all, but if that is the case, the libsonata config parser raises an error so this code will never be executed.